cbor/src/main/java/com/fasterxml/jackson/dataformat/cbor/CBORGenerator.java

```diff
@@ -61,7 +66,14 @@
          * Default value is <code>false</code> meaning that type tag will not be
          * written at the beginning of a new document.
          */
-        WRITE_TYPE_HEADER(false)
+        WRITE_TYPE_HEADER(false),
+
+        /**
+         * Feature that determines if an invalid surrogate encoding found in the
+         * incoming String should fail with an exception or silently be outputed
+         * as the Unicode 'REPLACEMENT CHARACTER' (U+FFFD)
+         */
```

Comment:
Could you add `@since` annotation (but see discussion on version to include in)

Reply:
Done